### PR TITLE
[Merged by Bors] - feat(Topology/Group): add `nhds_inv`

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Group/Basic.lean [hunk 1]
```diff
@@ -304,6 +304,10 @@ protected def Homeomorph.inv (G : Type*) [TopologicalSpace G] [InvolutiveInv G]
 lemma Homeomorph.coe_inv {G : Type*} [TopologicalSpace G] [InvolutiveInv G] [ContinuousInv G] :
     ⇑(Homeomorph.inv G) = Inv.inv := rfl
 
+@[to_additive]
+theorem nhds_inv (a : G) : 𝓝 a⁻¹ = (𝓝 a)⁻¹ :=
```
Comment (b-mehta) [inline Mathlib/Topology/Algebra/Group/Basic.lean:308]: Do you think this could be a simp lemma, the other way round?

Comment (urkud) [reply Mathlib/Topology/Algebra/Group/Basic.lean:308]: IDK. We rarely do computations with `nhds` filters and "pointwise" notation.

Comment (b-mehta) [reply Mathlib/Topology/Algebra/Group/Basic.lean:308]: True, but that feels like an argument to make it a simp lemma in the reverse direction: it makes the pointwise notation disappear easily

**Review by b-mehta — APPROVED** [review]
bors d+

**Comment by urkud** [comment]
We can experiment with `simp` attr later, adjusting the `GroupWithZero` version at the same time.
bors merge

**Pull request closed without merging**